CONTRIBUTING.md

```diff
@@ -44,9 +44,9 @@ Pull requests are the primary mechanism we use to change Cozy. GitHub itself has
 Fork the project on GitHub and [check out your copy locally][forking].
 
 ```
-$ git clone github.com/cozy/ogma.git
-$ cd ogma
-$ git remote add fork git://github.com/yourusername/ogma.git
+$ git clone github.com/cozy/Papillon.git
```

Comment:
```suggestion
$ git clone github.com/cozy/cozy-papillon.git
```

Reply:
Uh, I should have read this more carefully. Thank you for you vigilance!